source/common/formatter/substitution_formatter.cc

```diff
@@ -65,15 +65,15 @@ FormatterPtr SubstitutionFormatUtils::defaultSubstitutionFormatter() {
   return FormatterPtr{new FormatterImpl(DEFAULT_FORMAT, false)};
 }
 
-const absl::optional<std::string>
+const absl::optional<std::reference_wrapper<const std::string>>
```

Comment:
Let's add a comment here explaining why this is a reference_wrapper.

Reply:
Done!